interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -289,7 +290,7 @@ private void runTest(TestCases testCase) throws Exception {
 
   private class Tester extends AbstractInteropTest {
     @Override
-    protected ManagedChannel createChannel() {
```

Comment:
These methods were nice to use because they didn't take any args, and just picked some defaults.  Can you make it work the same way with Census?